test-lib.sh

```diff
@@ -50,6 +50,33 @@ function ct_enable_cleanup() {
   trap ct_cleanup EXIT SIGINT
 }
 
+function ct_check_envs_set {
```

Comment:
Some comment before this function whould be nice.

Reply:
True, will add. Thanks!